### PR TITLE
chore: redesign personal information page

### PR DESCRIPTION
**Pull request by lucyogutu**
Screenshot_1689148647

### CODE REVIEW

**Comment by lucyogutu** [comment]
Screenshot_1689152032

File: android/.project [hunk 1]
```diff
@@ -16,12 +16,12 @@
...
 		<filter>
-			<id>1631277708616</id>
+			<id>1689073643366</id>
 			<name></name>
 			<type>30</type>
 			<matcher>
 				<id>org.eclipse.core.resources.regexFilterMatcher</id>
-				<arguments>node_modules|.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
 			</matcher>
```
Comment (EugeneDevv) [inline android/.project:25]: Revert these changes

File: android/.settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-arguments=
+arguments=--init-script /home/lucyogutu/.config/Code/User/globalStorage/redhat.java/1.20.0/config_linux/org.eclipse.osgi/52/0/.cp/gradle/init/init.gradle --init-script /home/lucyogutu/.config/Code/User/globalStorage/redhat.java/1.20.0/config_linux/org.eclipse.osgi/52/0/.cp/gradle/protobuf/init.gradle
 auto.sync=false
 build.scans.enabled=false
 connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
```
Comment (EugeneDevv) [inline android/.settings/org.eclipse.buildship.core.prefs:4]: These as well

File: android/app/.project [hunk 3]
```diff
@@ -22,12 +22,12 @@
...
 		<filter>
-			<id>1631277708619</id>
+			<id>1689073643371</id>
 			<name></name>
 			<type>30</type>
 			<matcher>
 				<id>org.eclipse.core.resources.regexFilterMatcher</id>
-				<arguments>node_modules|.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
+				<arguments>node_modules|\.git|__CREATED_BY_JAVA_LANGUAGE_SERVER__</arguments>
 			</matcher>
```
Comment (EugeneDevv) [inline android/app/.project:31]: Same here

**Review by EugeneDevv — APPROVED** [review]

**Review by abiudrn — APPROVED** [review]
